keel-core/src/test/kotlin/com/netflix/spinnaker/keel/veto/UnhappyVetoTests.kt

```diff
@@ -101,20 +98,6 @@ class UnhappyVetoTests : JUnit5Minutests {
             that(response3.allowed).isFalse()
           }
         }
-
-        test("a happy resource should no longer be skipped ") {
```

Comment:
This flow was no longer present in `UnhappyVeto`, in fact `ResouceRepository` was unused there, so I removed the test.